packages/engine/bin/cli/src/experiment.rs

```diff
@@ -142,6 +143,7 @@ async fn run_experiment_with_manifest(
             }
             proto::EngineStatus::Errors(sim_id, errs) => {
                 error!("There were errors when running simulation [{sim_id}]: {errs:?}");
+                errored = true;
```

Comment:
If one simulation run failed, other simulation runs might still have succeeded, so I guess we would still want to persist the outputs of other simulation runs, but the CLI could exit with an error code.

Reply:
Thank you for clarifying!